client/src/components/sidebar/DarkModeButton.tsx

```diff
@@ -31,7 +31,7 @@ const DarkModeButton: React.FC<DarkModeButtonProps> = ({ collapsed }) => {
         <ToggleDarkModeButton color="inherit" onClick={toggleDarkMode}>
           {isDarkMode ? <LightModeIcon /> : <DarkModeIcon />}
           <IndividualComponentTypography collapsed={collapsed}>
-            {collapsed ? '' : isDarkMode ? 'Change to Light Mode' : 'Change to Dark Mode'}
+            {collapsed ? '' : isDarkMode ? 'Light Mode' : 'Dark Mode'}
```

Comment:
I might be tweaking but I think the `IndividualComponentTypography` component isn't actually using `collapsed`. Let's refactor this to be something like:

```suggestion
        {!collapsed && <IndividualComponentTypography
            {isDarkMode ? 'Light Mode' : 'Dark Mode'}
        </IndividualComponentTypography>}
```